---
content_type: page
description: This section provides an introduction to the lab and descriptions of
  the lab projects.
learning_resource_types:
- Labs
ocw_type: CourseSection
title: Labs
uid: 96637730-3e6e-8fb3-17f3-c255e0104a2b
---

Lab Introduction Lecture Notes ({{% resource_link b0311786-b363-df73-9508-7bdc851b4526 "PDF - 1.7 MB" %}})

Laboratory Project Part I
-------------------------

Day 1 (Lecture #3):  
Inoculation of 4 overnight cultures (16hr growth) of plasmid-bearing E. coli XL-1 blue.

Day 2 (Lecture #4):  
Harvesting of the bacteria by centrifugation and storage of the cell pellets at -80°C

Day 3 (Lecture #5):  
DNA preparation (4 samples)  
Absorption measurement at 260nm to determine the DNA concentration  
Sample preparation for automated DNA sequencing

Day 4 (a day before Lecture #6):  
Sample submission for DNA sequencing

Day 5 (Lecture #7):  
DNA restriction digest  
Agarose gel electrophoresis  
Hand-out of the sequencing results

**Your Job:**

*   Find as much information as possible about your clones.  
    Remember that the clones stem from an animal model. Why is the chicken an appropriate model for inner ear gene discovery? What are disadvantages of the chicken model?  
      
    Pick one or two clones that may have significance for inner ear function. What is known about these genes?
*   Document your experiments as detailed as possible starting with day1. A written lab report including a discussion of the results has to be submitted on or before Lecture #16.

**Plasmid DNA Preparation - Qiagen Method  
**  
See also [www.qiagen.com](http://www.qiagen.com) - QIAprep Miniprep Handbook

1.  Resuspend bacterial cell pellet in 250µl buffer P1 and transfer to a microcentrifuge tube.
2.  Have a timer ready - the next incubation step has to be exactly 4-5min!
3.  Add 250µl buffer P2, close the tube and immediately mix by 5-10 inversions.
4.  After exactly 4min, add 350µl buffer N3, close the tube and immediately mix by 5-10 inversions.
5.  Centrifuge for 10min at full speed in a tabletop microcentrifuge. The whitish pellet contains proteins and the bacterial genomic DNA.
6.  Transfer the supernatant into a spin (QIAprep) column. Do not transfer any material from the white pellet.
7.  Switch on the vacuum to draw the solution through the column (The plasmid DNA binds to the column material).
8.  Wash the column by adding 750 µl buffer PE.
9.  Transfer the column into a microcentrifuge tube and spin for 1min at full speed to dry the column material. (This removes any ethanol, which is part of buffer PE. Ethanol will interfere with subsequent manipulations of the DNA).
10.  Place the column in a clean microcentrifuge tube. Label the tube. Elute DNA by adding 70 µl of elution buffer (EB) into the center of the column material (do not touch the white column material though). Let sit for 1min and then centrifuge at full speed for 1min.

**Determination of the Plasmid DNA Concentration:**

1.  Dilute 5µl of each of your plasmid DNA 20-fold: This is a 1:20 dilution and means that you mix 95µl sterile MilliQ-filtered water with 5µl plasmid DNA solution.
2.  Your blank will be sterile MilliQ-filtered water. Your wavelength is 260nm.
3.  Blank the UV-spectrophotometer with MilliQ water. Careful, the Quartz cuvette is small, expensive, and it breaks when dropped on the floor!
4.  Exchange the water with your diluted sample and read sample.
5.  1 A260 unit of a DNA solution has a concentration of 50µg DNA / ml.
6.  DNA concentration in µg/ml = A260 reading \* dilution \* 50
7.  Remember, 500 µg/ml equals 500 ng/µl

**Sample Preparation for Sequencing:**

1.  The facility asks for 10µl DNA solution at a concentration of 200 ng/µl. In other words, they want 2µg of plasmid DNA.  
      
    If your DNA concentration is above 200 ng/µl, you need to dilute the appropriate amount.  
    If your DNA concentration is below 200 ng/µl but above 100 ng/µl, give them 10µl of your DNA solution (therefore you give them at least 1µg).  
    If your DNA concentration is below 100 ng/µl, your yield is too low for successful DNA sequencing.
2.  Label the tubes on top with your sample number, followed by a dash and a T for template (for example 5A-T or 3C-T).
3.  We will use a standard DNA sequencing primer at a concentration of 10 ng/µl - We will need 10µl of this oligonucleotide solution per sequence reaction. This solution will be provided. The sequencing primer will anneal upstream of the 5' cloning site (the EcoRI) of pAD-Gal4.  
    pAD-Gal4 forward seq 5' GGAATCACTACAGGGATG 3'

Laboratory Project Part II
--------------------------

Day 5 (Lecture #7):  
DNA restriction digest  
Agarose gel electrophoresis  
Hand-out of the sequencing results

**Your Job:**

*   A) Find as much information as possible about your clones.  
    Remember that the clones stem from an animal model. Why is the chicken an appropriate model for inner ear gene discovery? What are disadvantages of the chicken model?  
      
    Pick one or two clones that may have significance for inner ear function. What is known about these genes?
*   B) Document your experiments as detailed as possible starting with day1.  
    A written lab report including a discussion of the results has to be submitted on or before Lecture #16.

**DNA Restriction Digest  
**  
Each plasmid contains a cDNA insert that was cloned via EcoRI and XhoI. Consequently, a double-digest with the restriction endonucleases EcoRI and XhoI should release the insert from the plasmid.

To make pipetting easier, we will use an enzyme Master Mix that already contains the appropriate buffers and the two restriction enzymes.

1.  Aliquot 10µl of the master mix for each DNA to be analyzed into a reaction tube. Label the tubes.
2.  Add 10µl of each DNA solution to the Master Mix.
3.  Collect the 20µl of DNA and enzyme mixture on the bottom of the tube by brief centrifugation.
4.  Incubate at 37°C for 45-90 min.
5.  Add 7µl DNA loading buffer and analyze by Gel electrophoresis and comparison with Marker DNA fragments of known size.

**Master Mix (2x concentrated):**

> 280µl sterile MilliQ water  
> 80µl 10x buffer (New England Biolabs)  
> 20µl EcoRI enzyme (10-20U/µl)  
> 20µl XhoI enzyme (10-20U/µl)

**Agarose Gel Electrophoresis:**

1.  Wear gloves because the electrophoresis plasticware could be contaminated with the carcinogen Ethidiumbromide, that is used to visualize DNA.
2.  Boil 1g Agarose in 100ml Gel running buffer (microwave: 45sec, swirl, 20sec, swirl, 20sec). Let cool for 10min.
3.  Add 3µl of Ethidiumbromide and pour into assembled gel tray.
4.  Let gel solidify (15min).
5.  Load DNA samples (15-20µl/well).
6.  Run at 100V for 30-45min and visualize DNA fragments with UV light (wear eye protection!).
7.  Get a printout of your gel and determine the inserts' sizes by comparison with the markers.